ansible/roles/products/hand_e/cyberglove/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Ensures that shadow_hand_launcher directory exists
+  file:
+    path: "{{ shadow_hand_launcher_folder }}"
+    state: directory
+
+- name: Copy Cyberglove desktop icon
+  copy: 
+    src: files/cyberglove.png
+    dest: "{{ shadow_hand_launcher_folder }}/cyberglove.png"
+    mode: '664'
+
+- name: Create Cyberglove executable launch script
+  template:
+    src: templates/scripts/launch_cyberglove.j2
+    dest: "{{ shadow_hand_launcher_folder }}/launch_cyberglove.sh"
+    mode: '755'
+
+- name: Create Cyberglove desktop icon
+  template:
+    src: ../desktop-icons/templates/desktop_labels/desktop_icon.j2
```

Comment:
using file from different role. Seems really bad idea. You are breaking encapsulation of role. Please add big TODO here to fix it.

Reply:
Fixed in the last commit